frontend/src/views/main/utils/SlicingFunctionSelector.vue

```diff
@@ -1,20 +1,11 @@
 <template>
-    <div class="d-flex" :class="{w100: fullWidth}">
-        <v-select
-            clearable
-            :outlined="fullWidth"
-            class="slice-function-selector"
-            :label="label"
-            :value="value"
-            :items="slicingFunctions"
-            :item-text="extractName"
-            item-value="uuid"
-            :return-object="false"
-            @input="onInput"
-            :dense="fullWidth"
-            hide-details
-            :prepend-inner-icon="icon ? 'mdi-filter' : null"
-        ></v-select>
+    <div class="d-flex" :class="{ w100: fullWidth }">
+        <v-select clearable :outlined="fullWidth" class="slice-function-selector" :label="label" v-model="value" :items="[{
+            name: 'None',
```

Comment:
I don't understand why we have a `None` argument being passed

Reply:
I and @jmsquare took the decision to display the default value for each input (if it has a default value). 
In the case of the Slicing Function, it was displaying a selector with no option selected, so it was not very clear if it was a required field or not.